deploy/sre-pruning/images-pre-4.6/00-pruning-images.CronJob.yaml

```diff
@@ -1,4 +1,3 @@
----
```

Comment:
Should the permissions for the SA be split out too?

Reply:
Great catch, I didn't think of that. I've broken out the ClusterRoleBinding to sit within the sub-dir now.